README.md

```diff
@@ -55,6 +55,7 @@ Supported keys of the map are:
 ## License
 
 Copyright © 2016 Coda Hale
+Copyright © 2017 Sentry
```

Comment:
Sorry about that! I meant to add a description (and just did). I actually originally thought this PR was going against my own repo.